packages/node_modules/samples/browser-plugin-meetings/test/wdio/spec/mediatest.js

```diff
@@ -65,7 +65,7 @@ describe('samples', () => {
         browserSpock.click('#addMedia');
       });
 
-      it('spock answers the call', () => {
+      flaky(it, process.env.SKIP_FLAKY_TESTS)('spock answers the call', () => {
```

Comment:
I think this needs to be refactored so that the answer button isn't hidden.
It's half the reason the test fails.
However, the test is flaky locally on firefox due to a h264 issue, that I believe has been fixed

Reply:
Ran the sample locally, it's fine
Disregard my comment

Reply:
@arun3528